admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
Agreed and done. Although in this case we still need the capability strings. I've set `is_level_admin()` to return a boolean just like `is_super_admin()`, this way you can mangage "backup" capabilities per case.

> But I think it can easily become a mess if using different capabilities/checks depending on the type of the site, so I think just making sure all admins get manage_restrictions on plugin activation is better. There is already code in place to add callbacks when updating the plugin as well, so existing users of the plugin will get migrated.

True, but one problem there, we don't know with single installations that the `administrator` role is the highest. I know some people create their own "god" roles etc. for that stuff. That's why I'd prefer it this way. Now we know in single installations that only the users with the custom capability OR `delete_users` (aka, single site super admins) can access.

When a user can access capability management there is no way to prevent them from elevating themselves. This is simply something a site admin should manage correctly imo.

Please check the commit and let me know!